capture-sdk/sdk/src/main/res/layout/gc_item_format_header.xml

```diff
@@ -3,15 +3,17 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@color/gc_supported_formats_activity_background"
+    android:background="?attr/backgroundColor"
     android:orientation="vertical">
 
     <net.gini.android.capture.internal.ui.CustomFontTextView
         android:id="@+id/gc_supported_formats_item_header"
-        style="@style/GiniCaptureTheme.Help.SupportedFormats.Item.Header.TextStyle"
+        style="@style/GiniCaptureTheme.Typography.Subtitle2"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/gc_supported_formats_item_header_margin"
+        android:layout_marginStart="@dimen/gc_supported_formats_item_header_margin"
```

Comment:
[Request] Please replace it with one of the size class dimensions. I think `large_24` would fit.